bot/cogs/ext/commands/anime.py

```diff
@@ -1,21 +1,21 @@
 """Father, forgive me for these sins."""
 
 import asyncio
+from typing import List, Union
```

Comment:
From X import Y imports should be placed on the second block

1st block is for general imports
2nd block is for from X import Y
3rd block is for custom Pidroid imports